src/main/java/me/partlysanestudios/partlysaneskies/help/DiscordCommand.java

```diff
@@ -40,17 +40,17 @@ public int compareTo(ICommand o) {
 
     @Override
     public String getCommandName() {
-        return "discord";
+        return "pssdiscord";
```

Comment:
that is better thank you, I will have to double check that the skytils repo doesn't use it thought